### ISSUE TITLE
Enhancement: Have the Animation Offset Editor menu play Artistic Expression in the background

### ISSUE THREAD

**Issue by PSI3x** [FunkinCrew/Funkin#3774]
### Issue Checklist

- [ ] I have properly named my enhancement
- [ ] I have checked the Issues/Discussions pages to see if my enhancement has already been suggested

### What is your suggestion, and why should it be implemented?

Personally, I believe the title screen theme ("Freaky Menu") does not fit with the animation editor. 
[The calmer chart editor theme](https://www.youtube.com/watch?v=yFHbQFH09Io) would fit this interface much better in my opinion.

**Comment by Hundrec** [comment]
This would be a logical change following Kolosaki's Stage Editor adopting Artistic Expression.
Though it would be cool to see every editor have a unique theme eventually!

**Comment by JustKolosaki** [comment]
> Though it would be cool to see every editor have a unique theme eventually!

Yeah especially because internally it's called *chart*EditorLoop..........

**Comment by trayfellow** [comment]
I think the animation editor (which should be renamed to character editor at this point) will use the Artistic Expression theme once it gets a proper revamp like the Chart&Stage editor

**Comment by EliteMasterEric** [comment]
I like this but it might go better with a proper revamp to the entire editor, either in #3935 or a separate overhaul.
